packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -352,7 +352,7 @@ export const FormControl = forwardRef<HTMLInputElement, PropsWithChildren<FormCo
   const Icon = icon ? (
     <Flex
       as={'span'}
-      title='A slug is a human-readable ID that must be unique.  It’s often used in URLs.'
+      title={t(localizationKeys('formFieldHintText__slug'))}
```

Comment:
I think this is better 

```suggestion
      title={t(localizationKeys('formFieldIconText__organizationSlug'))}
```

Reply:
If we want to reuse this in other places, `hintText__slug` is more generic and appropriate than tying this specific to organizations. The text isn't mentioning Organizations